web/annotation_layer_builder.css

```diff
@@ -26,11 +27,16 @@
   height: 100%;
 }
 
-.annotationLayer .linkAnnotation > a /* -ms-a */  {
+.annotationLayer .linkAnnotation > a /* -ms-a */ {
   background: url("data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7") 0 0 repeat;
 }
 
-.annotationLayer .linkAnnotation > a:hover {
+.annotationLayer .buttonWidgetAnnotation.pushButton > a /* -ms-a */ {
```

Comment:
I'm assuming that using the following format didn't work with the CSS preprocessor?
```CSS
.annotationLayer .linkAnnotation > a /* -ms-a */,
.annotationLayer .buttonWidgetAnnotation.pushButton > a /* -ms-a */ {
```

Anyway, I think that this is actually yet another reason to explicitly remove support for IE10 as well (in addition to IE9), since then we'd no longer need these hacks[1] :-)

---
[1] Given issue #3579.

Reply:
Yes, I think it doesn't support that, but indeed, we should get rid of that soon anyway, hopefully.